website/content/docs/k8s/installation/install-cli.mdx

```diff
@@ -1,23 +1,44 @@
 ---
 layout: docs
-page_title: Installing the Consul K8s CLI
+page_title: Install Consul from the Consul K8s CLI
 description: >-
-  Consul K8s CLI is a tool for quickly installing and interacting with Consul on Kubernetes.
+  This topic describes how to install Consul on Kubernetes uisng the Consul K8s CLI tool.
 ---
 
-# Installing the Consul K8s CLI 
 
-Consul K8s CLI is a tool for quickly installing and interacting with Consul on Kubernetes. Ensure that you are installing the correct version of the CLI for your Consul on Kubernetes deployment, as the CLI and the control plane are version dependent. 
+# Install Consul on Kubernetes from Consul K8s CLI 
+
+This topic describes how to install Consul on Kubernetes using the Consul K8s CLI tool. The Consul K8s CLI tool enables you to quickly install and interact with Consul on Kubernetes. Use the Consul K8s CLI tool to install Consul on Kubernetes if you are deploying a single cluster. We recommend using the [Helm chart installation method](/docs/k8s/installation/install) if you are installing Consul on Kubernetes for multi-cluster deployments that involve cross-partition or cross datacenter communication.
```

Comment:
> "if you are installing Consul on Kubernetes for multi-cluster deployments that involve cross-partition or cross datacenter communication."

Why is that?

Reply:
@t-eckert I think this is prior to us adding multi-dc support with the CLI. I believe at the point these docs were written, we did not test the CLI for Admin Partitions, WanFed, or Cluster Peering. If you feel this should be revised please provide a suggestion. Thanks!

Reply:
Ah, gotcha! We can leave this in for now. We should verify that the CLI works for these cases, though I'm fairly certain it does. I'd like to add acceptance tests for it, but our acceptance testing is already getting slow.